Sections/US-States/FL/GPP Extension: IAB Privacy’s Florida Privacy Technical Specification.md

```diff
@@ -0,0 +1,190 @@
+<h1 id="gpp-extension-iab-privacy-s-florida-privacy-technical-specification">GPP Extension: IAB Privacy’s Florida Privacy Technical Specification</h1>
+<h2 id="about-this-document">About this document</h2>
+<p>The global standard <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform">GPP</a> defines a way for local standards to &quot;plug-in&quot; into the existing mechanics defined by GPP and the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Core/CMP%20API%20Specification.md">GPP client side API</a>. This document outlines the technical specification for using the GPP specifications with the IAB Privacy Multi-State Privacy Agreement legal requirements.</p>
+
+<h3>Version History&nbsp;</h3>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Date</strong></td>
+<td><strong>Version</strong></td>
+<td><strong>Comments</strong></td>
+</tr>
+
+<tr>
+<td>May 2024</td>
+<td>1.0</td>
+<td>Version 1.0 released</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+<h2>Florida Section</h2>
+<p>The Florida Privacy String consists of the following components. Users should employ the Florida Privacy String only if they have determined the Florida Digital Bill of Rights, Fla. Stat. § 501.701 et seq., applies to their processing of a consumer’s personal data.</p>
+<h3>Summary</h3>
+<div>
+  <table>
+    <tbody>
+      <tr>
+        <td>
+          <strong>Type</strong>
+        </td>
+        <td>
+          <strong>Value</strong>
+        </td>
+        <td>
+          <strong>Description</strong>
+        </td>
+      </tr>
+      <tr>
+        <td>GPP Section ID</td>
+        <td>13</td>
+        <td>The Florida Section is registered as Section ID 13 under the GPP.</td>
+      </tr>
+      <tr>
+        <td>Client side API prefix</td>
+        <td>usfl</td>
+        <td>The Florida Privacy Section is registered with client side API prefix “usfl” in the GPP Client Side API.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+<h3>Section encoding</h3>
+<h4>Core Segment</h4>
+<p>The core sub-section must always be present. Where terms are capitalized in the ‘description’ field they are defined terms in Fla. Stat. § 501.702. It consists of the following fields:</p>
+<div>
+  <table>
+    <thead>
+      <tr>
+        <th>
+          <p>Field name</p>
+        </th>
+        <th>
+          <p>GPP Field Type</p>
+        </th>
+        <th>
+          <p>Description</p>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">Version</span>
+        </td>
+        <td>Int(6)</td>
+        <td>
+          <span style="color:rgb(36, 41, 47);">The version of this section specification used to encode the string.</span>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <span style="color:rgb(36, 41, 47);">ProcessingNotice</span>
```

Comment:
suggest adding a binary for  Fla. Stat. § 501.701 et seq applies

Reply:
is this something that section id (13) for Florida can achieve with it's use denoting applicability? Or is this for a different item around identification of applicability of Florida to a sender?

Reply:
it is risky to the publisher to say the law applies when they may not be sure, this and the above comment are two solutions to the same problem